### DISCUSSION TITLE
Grid Layout for Watch tab

### DISCUSSION THREAD

**Discussion by zerodrek** [javikr/sofatime#50] [Ideas]
It would be nice to have the option for a grid style layout for the watch tab. Something similar to this:
![grid layout](https://github.com/javikr/sofatime/assets/2582474/afe35d82-d4c9-464c-9c6e-3f118e76144f).

**Comment by javikr** [comment]
Hi there! Sorry for the late response. 

Yes, definitively it's a good idea. I will analyse it to add to our roadmap. Thanks!
